core/assets/bundles/bundle_fr.properties

```
@@ -825,16 +825,16 @@ threat.high = Grande
 threat.extreme = Extrême
 threat.eradication = Éradication
 
-difficulty.casual = Casual
-difficulty.easy = Easy
+difficulty.casual = Paisible
+difficulty.easy = Facile
 difficulty.normal = Normal
-difficulty.hard = Hard
-difficulty.eradication = Eradication
+difficulty.hard = Difficile
+difficulty.eradication = Éradication
 
-difficulty.enemyHealthMultiplier = Enemy Health: {0}
-difficulty.enemySpawnMultiplier = Enemy Amount: {0}
-difficulty.waveTimeMultiplier = Wave Timer: {0}
-difficulty.nomodifiers = [lightgray](No modifiers)
+difficulty.enemyHealthMultiplier = Vie des ennemies : {0}
+difficulty.enemySpawnMultiplier = Nombre d'ennemies : {0}
+difficulty.waveTimeMultiplier = Espacement des Vagues : {0}
```

```suggestion
difficulty.enemyHealthMultiplier = Vie des ennemis: {0}
difficulty.enemySpawnMultiplier = Nombre d'ennemis: {0}
difficulty.waveTimeMultiplier = Espacement des Vagues: {0}
```
Il n'y a pas de "e" à la fin de ennemi.